networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -193,7 +193,7 @@ spec:
 ----
 <1> The name for the configuration object.
 <2> A node selector that specifies the nodes to apply the node network configuration policy to.
-<3> The name for the additional network from which traffic is forwarded to the OVS bridge. This additional network must match the name of the additional network that you define in the `netAttachDefName` field of a `NetworkAttachmentDefinition` object.
+<3> The name for the additional network from which traffic is forwarded to the OVS bridge. This additional network must match the name of the `metadata.name` field of the `NetworkAttachmentDefinition` object that defines the OVN-Kubernetes additional network.
```

Comment:
this is wrong, it's not the `metadata.name` attribute it must match - it is the `spec.config.name` attribute.

Reply:
@maiqueb, that makes a lot more sense; Not sure why this is causing me such trouble. I'll update these instances, thanks!